python/res/job_queue/ert_script.py

```diff
@@ -112,9 +111,21 @@ def outputStackTrace(self, error=None):
     def loadScriptFromFile(path):
         """ @rtype: type ErtScript """
         try:
-            m = imp.load_source("ErtScriptModule_%d" % ErtScript.__module_count, path)
+            module_name = "ErtScriptModule_%d" % ErtScript.__module_count
+            if sys.version_info[0] == 3 and sys.version_info[1] >= 5:
```

Comment:
I think the more standard way of writing this is `sys.version_info[:2] >= (3, 5)`. Feel that is slightly easier to read... Also, `elif` but no `else` is bad!

How about
```python
py_version = sys.version_info[:2]
if py_version == (2, 7):
    # Python 2.7 stuff
elif (3, 0) <= py_version < (3, 5):
   # Python 3 pre 3.5 stuff
elif (3, 5) <= py_version:
   # Python 3.5 and later stuff
else:
    raise Exception("Invalid Python verson {}:".format(py_version))
```

Reply:
Agreed!